scripts/depcheck.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+DEPCHECK_OUTPUT=`depcheck`
+
+if [[ `echo $DEPCHECK_OUTPUT | grep Missing | wc -l` -gt 0 ]]; then
+    echo "Found some missing dependencies."
+    echo "$DEPCHECK_OUTPUT"
+    exit 1
+    echo "`yarn add` the missing dependencies and re-commit package.json."
```

Comment:
I think this line will never print since we `exit 1` right above it

Reply:
ah dumb mistake, thank you!